src/stb_vorbis.h

```diff
@@ -5656,6 +5657,11 @@ int stb_vorbis_get_samples_float_interleaved(stb_vorbis *f, int channels, float
          break;
    }
    f->current_playback_loc += n;
+   if(f->current_playback_loc > lgs && lgs > 0 && lgs != UINT_MAX) {
```

Comment:
Why not to just use the `SAMPLE_unknown` directly? It's private code, and you are free to use that.
```cpp
if(f->current_playback_loc > lgs && lgs > 0 && lgs != SAMPLE_unknown) {
```

Reply:
Adjusted as request : https://github.com/icculus/SDL_sound/compare/fef554e25e79aef3dd24fa964e219f9a5b20a309..945fbbe3a09dd1ae043c2a6b02c9e622cd24f346

I stand-by what I said that stb_vorbis using ints directly is terrible and it should use size_t and SIZE_MAX instead or any other regular c type that is appropriate or typedef things to make sense instead of using naked int/uint mixed around...